crashreports/internal/win32/CrashReportContext.cpp

```diff
@@ -0,0 +1,115 @@
+#include <locale>
+#include <codecvt>
+#include <sstream>
+#include "client/windows/handler/exception_handler.h"
+#include "CrashReportContext.h"
+
+namespace {
+	bool uploadReport(
+		const wchar_t* dump_path,
+		const wchar_t* minidump_id,
+		void* context,
+		EXCEPTION_POINTERS* exinfo,
+		MDRawAssertionInfo* assertion,
+		bool succeeded)
+	{
```

Comment:
I figured out that this function executes in the context of a crash, as also the makeCommand and send member functions, and therefore do unusual things to avoid memory allocation.  But these three should contain comments, to make that clearer.

Reply:
Can you comment, or point to documentation, explaining the call signature that this function must have?

For instance I don't understand "succeeded" just reading this.